docker-compose.yml

```diff
@@ -156,6 +156,28 @@ services:
       - "traefik.frontend.rule=${TRAEFIK_FRONTEND_RULE:-}PathPrefix:/bety/"
       - "traefik.backend=bety"
 
+  # ----------------------------------------------------------------------
+  # Pecan-RStudio
+  # ----------------------------------------------------------------------
+  rstudio:
+    image: pecan/base:1.7.0
```

Comment:
Use same syntax as other pecan containers to make sure we have latest etc options

Reply:
Update to variable for version (in env.example as well)